### PR TITLE
update profile with arc and arc_ecto, profile templates and route

### PR DESCRIPTION
**Pull request by hendri-tobing**
Update the following: 
* Profile's avatar field with arc and arc_ecto (initial work on #571 ), 
* Profile templates, 
* Profile route.

When testing with arc_ecto, there are some problems:
* It does not work with Dialyzer, [reference: stackoverflow](https://stackoverflow.com/questions/49511255/typespecs-for-arc-ecto-changeset) 
* It insert DateTime on updated_at field with microseconds. This is causing many tests fail, [the issue: here](https://github.com/stavro/arc_ecto/issues/96)

For a temporary solution, I forked arc_ecto and update the necessary things to pass the above issues. I add this on the `mix.exs`

`{:arc_ecto, git: "https://github.com/hendri-tobing/arc_ecto.git"}`

### CODE REVIEW

**Review by wintermeyer — DISMISSED** [review]
I'm not the right person to judge this PR but I approve it to keep things going. Obviously the self hosted arc_ecto is suboptimal. 

@riverrun Please share your thoughts when you are ready.

File: lib/vutuv/biographies/profile.ex [hunk 1]
```diff
@@ -27,6 +28,7 @@ defmodule Vutuv.Biographies.Profile do
           updated_at: DateTime.t()
         }
 
+  @timestamps_opts [type: :naive_datetime, usec: false]
```
Comment (riverrun) [inline lib/vutuv/biographies/profile.ex]: Why do we need this?

**Review by riverrun — COMMENTED** [review]
Just one question.

Also, the travis build is failing with this error:

warning: function validate_file/1 is unused
  lib/vutuv_web/uploaders/avatar.ex:33
Compilation failed due to warnings while using the --warnings-as-errors option
The command "mix compile --warnings-as-errors" exited with 1.

File: lib/vutuv/biographies/profile.ex (hunk 1 above)
Comment (hendri-tobing) [reply lib/vutuv/biographies/profile.ex]: No we don't need it. Thank you for the review.

**Comment by riverrun** [comment]
Looks good.

**Review by riverrun — APPROVED** [review]

**Pull request merged**